Sorting/BucketSort.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream> 
+#include <algorithm> 
+#include <vector> 
+using namespace std; 
+static void BucketSort(int* data, int count) {
+	int minValue = data[0];
+	int maxValue = data[0];
+	for (int i = 1; i < count; i++)
+	{
+		if (data[i] > maxValue)
+			maxValue = data[i];
+		if (data[i] < minValue)
+			minValue = data[i];
+	}
+	int bucketLength = maxValue - minValue + 1;
+	vector<int>* bucket = new vector<int>[bucketLength];
+  for (int i = 0; i < bucketLength; i++)
+	{
+		bucket[i] = vector<int>();}
+for (int i = 0; i < count; i++)
```

Comment:
Could you apply same formatting rules through out the file?
Can you add comments to guide the user?

Reply:
Could you apply same formatting rules through out the file?